README.md

```diff
@@ -18,7 +28,39 @@ Deployment utility for Educandu projects
 | wait | Whether or not to wait for service stability | boolean | no (defaults to false) |
 
 Note:
-`container-env` can be set multiple times, per each environment variable needed for the deployed container
+
+* `container-env` can be set multiple times, per each environment variable needed for the deployed container
+
+## Command: edge
+
+Updates code and environment variables of an AWS Lambda@Edge function, publishes a new version for the
+function, and updates a CloudFront distribution that is using this function to the published version.
+
+### Usage
+
+~~~
+$ ./node index.js edge --access-key 487nct72tc4tdjgbj ...
+~~~
+
+### Arguments
+
+| Name | Description | Type | Required |
+| --- | --- | --- | --- |
+| access-key | AWS access key | string | yes |
+| secret-key | AWS secret key | string | yes |
+| lambda-env | Environment variable pair to be available to the deployed function, format `name=value` | array of string | no |
+| lambda-env-inject | Optional: Path of the JS file inside the zip file to directly inject environment variables | array of string | no |
```

Comment:
Tiny thing: it's already marked as "Required: no", you could drop the "Optional: "

Reply:
Thanks, done